python/pyspark/ml/feature.py

```diff
@@ -1163,9 +1184,11 @@ class QuantileDiscretizer(JavaEstimator, HasInputCol, HasOutputCol, JavaMLReadab
 
     >>> df = spark.createDataFrame([(0.1,), (0.4,), (1.2,), (1.5,)], ["values"])
     >>> qds = QuantileDiscretizer(numBuckets=2,
-    ...     inputCol="values", outputCol="buckets", relativeError=0.01)
+    ...     inputCol="values", outputCol="buckets", relativeError=0.01, handleInvalid="error")
     >>> qds.getRelativeError()
     0.01
+    >>> qds.getHandleInvalid()
```

Comment:
We didn't add anything to the doctest of bucketizer. Actually, I think it would be nice in both places to set `handleInvalid='skip'` and then add an invalid value to the example data. That way we can show what we mean by invalid and prove that it works.